--- pandoc.cabal,v	Mon Oct 28 22:33:19 2019
+++ pandoc.cabal	Mon Oct 28 22:32:14 2019
@@ -623,7 +623,7 @@
      hs-source-dirs: prelude
      other-modules:  Prelude
      build-depends:  base-compat >= 0.9
-  ghc-options:   -rtsopts -with-rtsopts=-K16m -Wall -fno-warn-unused-do-bind -threaded
+  ghc-options:   -rtsopts -with-rtsopts=-K16m -Wall -fno-warn-unused-do-bind
   if flag(static)
     ld-options:     -static
   default-language: Haskell2010
